samples/graph-chat-lifecycle/nodejs/Helpers/ChatHelper.js

```diff
@@ -0,0 +1,198 @@
+              user4Id:users.value[3].id,
+              user5Title:users.value[4].displayName,
+              user5Id:users.value[4].id,
+              user6Title:users.value[5].displayName,
+              user6Id:users.value[5].id
+            }
+          }); 
+         var card = CardFactory.adaptiveCard(cardPayload);    
+          res.send(card);
+        } catch (error) {
+            console.log(error);
+        }
+    }     
+}
+
+createGroupChat = (req,res)=>{
+      
+  const userID = req.body.users.split(",");
+  const title = req.body.title;
+  token = req.body.token;
+    const client = Client.init({
+      authProvider: (done) => {
+      done(null, token); // First parameter takes an error if you can't get an access token.
+      }
+      });
+
+      createChat();
+
+    async function createChat()
+    {
+      try {
+          const chat = {
+          chatType: 'group',
+          topic: title,
+          members: [
+            {
+              '@odata.type': '#microsoft.graph.aadUserConversationMember',
+              roles: ['owner'],
+              'user@odata.bind': 'https://graph.microsoft.com/v1.0/users('+'\''+userID[0]+'\''+')'              
+            },
+            {
+              '@odata.type': '#microsoft.graph.aadUserConversationMember',
+              roles: ['owner'],
+              'user@odata.bind': 'https://graph.microsoft.com/v1.0/users('+'\''+req.body.userId+'\''+')'
+            }
+          ]
+        };
+        var response = await client.api('/chats')
+          .post(chat);
+
+          if(userID.length ==2)
+          {
+             await addMembersWithHistory(client,response,userID);
+             await deleteMember(client,response,userID);
+          }
+          else if(userID.length ==3)
+          {
+            await addMembersWithHistory(client,response,userID);
+            await addMembersWithoutHistory(client,response,userID);
+            await deleteMember(client,response,userID);
+          }
+          else if(userID.length >=4)
+          {
+            await addMembersWithHistory(client,response,userID);
+            await addMembersWithoutHistory(client,response,userID);
+            await addMembersWithNoOfDays(client,response,userID);
+            await deleteMember(client,response);
+          }          
+              
+          //Adding Polly App to chat
+          const teamsAppInstallation = {
+              'teamsApp@odata.bind':'https://graph.microsoft.com/v1.0/appCatalogs/teamsApps/1542629c-01b3-4a6d-8f76-1938b779e48d'
+              };
+              
+          await client.api('/chats/'+response.id+'/installedApps')
+                .post(teamsAppInstallation);
+           
+
+          //Adding Polly app as Tab to chat      
+          const teamsTab = {
+                displayName: 'Polly',
+                'teamsApp@odata.bind': 'https://graph.microsoft.com/v1.0/appCatalogs/teamsApps/1542629c-01b3-4a6d-8f76-1938b779e48d',
```

Comment:
Please read polly App Id from configuration and mention about polly App Id in readMe